android/media/audio/record_audio/RecordAudio/Activity1.cs

```diff
@@ -28,7 +29,17 @@ protected override void OnCreate (Bundle bundle)
             _start = FindViewById<Button> (Resource.Id.start);
             _stop = FindViewById<Button> (Resource.Id.stop);
                     
-            string path = "/sdcard/test.3gpp";
+			string path = $"{Android.OS.Environment.ExternalStorageDirectory.AbsolutePath}/test.3gpp";
+
+			//Explicit permissions
+			if (Build.VERSION.SdkInt > BuildVersionCodes.M)
+			{
+				if (CheckSelfPermission(Manifest.Permission.WriteExternalStorage) != Android.Content.PM.Permission.Granted
+				|| CheckSelfPermission(Manifest.Permission.RecordAudio) != Android.Content.PM.Permission.Granted)
+				{
+					RequestPermissions(new[] { Manifest.Permission.WriteExternalStorage, Manifest.Permission.RecordAudio }, 0);
```

Comment:
Before requesting permissions, should display the rationale for for the permissions with. Use ShouldShowRequestPermissionRationale() to see if the rationale should be displayed.